### PR TITLE
Reconcile docs

### PR DESCRIPTION
**Pull request by cliffckerr**
### Brief description
Updates the FPsim docs to use PyData theme, in line with [Covasim](http://covasim.org) and [HPVsim](http://hpvsim.org).

Old landing page:
image

New landing page:
image

Old API page:
image

New API page:
image

### Type(s) of change
- [ ] Bugfix
- [x] Refactor
- [ ] New feature
- [ ] Other

### Checklist
- My code follows the [style guide](https://github.com/amath-idm/styleguide)
  - [ ] Yes
  - [x] N/A
- I've commented my code
  - [ ] Yes
  - [x] N/A
- I've incremented the version number
  - [ ] Yes
  - [x] N/A
- I've updated the changelog
  - [ ] Yes
  - [x] N/A
- I've added tests and checked code coverage
  - [ ] Yes
  - [x] N/A

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -1,189 +1,133 @@
...
-with open(filename) as f: # Read existing file
-    lines = f.readlines()
-lines[0] = "API reference\n" # Blast away the existing heading and replace with this
-lines[1] = "=============\n" # Ensure the heading is the right length
-with open(filename, "w") as f: # Write new file
-    f.writelines(lines)
+# -- Project information -----------------------------------------------------
 
+project = 'FPsim'
+copyright = f'2020 - {sc.now().year}, Bill & Melinda Gates Foundation. All rights reserved.\nThese docs were built for FPsim version {fp.__version__}.\n'
```
Comment (JSchripsema-IDM) [inline docs/conf.py:19]: Removing the period at the end of this line will get rid of the double period showing in the footer currently.

**Review by JSchripsema-IDM — CHANGES_REQUESTED** [review]
Can you update the theme_overrides.css file to use one of the blues specified in our logo usage guide for link text? https://bmgf.sharepoint.com/sites/IDMOD/Shared%20Documents/Forms/AllItems.aspx?noAuthRedirect=1&id=%2Fsites%2FIDMOD%2FShared%20Documents%2FTemplates%20and%20Logos%2FIDM%20Logo%2FIDM%20Logo%20Usage%20Guide%2Epdf&viewid=acb933e7%2D35e7%2D459d%2Dba31%2D1c1bff1595fb&parent=%2Fsites%2FIDMOD%2FShared%20Documents%2FTemplates%20and%20Logos%2FIDM%20Logo

**Review by JSchripsema-IDM — APPROVED** [review]
LGTM

**Comment by cliffckerr** [comment]
Thanks @JSchripsema-IDM! Yes the link color is a little complicated ... it doesn't seem to be being set in `theme_overrides.css`. In fact we might be able to delete most/all of that file :D As you probably saw I made a new issue for changing the link color.

**Pull request merged**